src/psyclone/psyir/nodes/operation.py

```diff
@@ -101,9 +101,9 @@ def operator(self):
         Return the operator.
 
         :returns: Enumerated type capturing the operator.
-        :rtype: :py:class:`psyclone.psyir.nodes.UnaryOperation.Operator` or \
-                :py:class:`psyclone.psyir.nodes.BinaryOperation.Operator` or \
-                :py:class:`psyclone.psyir.nodes.NaryOperation.Operator`
+        :rtype: Union[
+            :py:class:`psyclone.psyir.nodes.UnaryOperation.Operator`
+            :py:class:`psyclone.psyir.nodes.BinaryOperation.Operator`]
```

Comment:
This made me realise that we no longer have NaryOperation but a quick grep revealed a few docstrings:
```
psyir/backend/fortran.py:            :py:class:`psyclone.psyir.nodes.NaryOperation` or \
psyir/nodes/operation.py:        :py:class:`psyclone.psyir.nodes.NaryOperation.Operator`
psyir/nodes/operation.py:                :py:class:`psyclone.psyir.nodes.NaryOperation.Operator`
psyir/transformations/intrinsics/minormax2code_trans.py:        must be a MIN or MAX BinaryOperation or NaryOperation. The
psyir/transformations/intrinsics/minormax2code_trans.py:            :py:class:`psyclone.psyir.nodes.NaryOperation`
```
would you mind updating them too?

Reply:
Done